hack/kustomization/whereabouts/kustomization.yaml

```diff
@@ -7,5 +7,7 @@ resources:
 - ./whereabouts.cni.cncf.io_ippools.yaml
 - ./whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml
 
-transformers:
-  - patch-ip-reconciler-job.yaml
+images:
+  - name: ghcr.io/k8snetworkplumbingwg/whereabouts
+    newName: ghcr.io/k8snetworkplumbingwg/whereabouts
```

Comment:
Can we remove newName or it is a must to keep it even that it is the same ?
beside that lgtm, thanks